django_socio_grpc/management/commands/grpcrunaioserver.py

```diff
@@ -93,7 +95,7 @@ def inner_run(self, *args, **options):
         # to be raised in the child process, raise it now.
         # ------------------------------------------------------------------------
         autoreload.raise_last_exception()
-        logger.info('"Performing system checks...\n\n')
+        logger.info('"Performing system checks...\n\n', True)
```

Comment:
logger.info('"Performing system checks...\n\n', emit_to_server=True) to make it easier to understand